virtme_ng/run.py

```diff
@@ -912,6 +912,8 @@ def _get_virtme_arch(self, args):
         if args.arch is not None:
             if args.arch not in ARCH_MAPPING:
                 arg_fail(f"unsupported architecture: {args.arch}")
+            if args.root is None:
+                sys.stderr.write("WARNING: --arch used without --root." + "\n")
```

Comment:
I'm wondering if we should just exit with an explicit error when `args.arch != get_host_arch()`, since we're going to fail anyway with obscure errors like #233

Reply:
Thank you for the review!

That makes sense. I didn't do that because there were maybe weird use-cases I didn't think about that. But if any, we can always fix those later on.